### PR TITLE
Salt PR Day 6

### PR DESCRIPTION
**Pull request by Parzivus**
Mostly minor changes. If the other PR branch goes through first, I can update this one.

### CODE REVIEW

File: script/Arcueid/Day 6/06_00_ARC06_4.txt [hunk 1]
```diff
@@ -240,10 +240,10 @@ How did my dear Nii-san,
 [sha:60b3d306f1170f0608bc363797cce938ef74db2b]{
 -- Page 339, Offset 14757.
 -- ……はあ。そんな下らない事を堂々と口にしないでください。私は何とも思いませんが、他の人が聞けば笑いものになるだけですから」
-...Sigh.
+...Hah.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 6/06_00_ARC06_4.txt:243]: Whatever we do with the others, here too

File: script/Ciel/Day 6/06_01_CIEL06_1.txt [hunk 2]
```diff
@@ -116,13 +116,14 @@ There is no greater reward for an insignificant person such as myself―――
 [sha:921d7ad03ca3d02ca063fdb60aef17885b4a4c52]{
 -- Page 8, Offset 13172.
 -- 思わず上半身が跳ねあがる。
-I reflexively snap bolt upright.
+// Maybe "snap" or "bolt" but not both?
```
Comment (rschlaikjer) [inline script/Ciel/Day 6/06_01_CIEL06_1.txt:119]: Assuming everything from here is just re-exports

File: script/Arcueid/Day 6/06_00_ARC06_1_1.txt [hunk 3]
```diff
@@ -83,7 +83,7 @@ Yes, it seems that we have a lot to catch up on, don't you think?"
 -- Page 56, Offset 14057.
 -- アレか。カエルを前にした蛇とか、そういうヤツか。
 // Not entirely sure what アレか is referring to.
-She must be. I'm like a deer caught in the headlights.
+She must be. I'm like a deer caught in headlights.
```
[the remark this answers is not in the document]
Comment (Parzivus) [reply script/Arcueid/Day 6/06_00_ARC06_1_1.txt:86]: Huh, I've only ever heard the latter. We can keep as-is then

File: script/Arcueid/Day 6/06_00_ARC06_1_3.txt [hunk 4]
```diff
@@ -152,7 +153,8 @@ After clearing her throat, Akiha exhales deeply and composes herself.
 [sha:374cf8f6c3447960c0e5dbf55a78a4c78f48f870]{
 -- Page 122, Offset 14220.
 -- 「兄さん。言っておきますが、そんな気持ちのこもった言葉では誤魔化されません」
-"Nii-san. I'll warn you now, you cannot fool me with such honeyed words."
+// Not sure how much we care about "no contractions" on Akiha. This could be "I will" or stay as "I'll" as well.
```
[the remark this answers is not in the document]
Comment (Parzivus) [reply script/Arcueid/Day 6/06_00_ARC06_1_3.txt:156]: It ends up sounding a little awkward if she _never_ uses contractions, yeah.

File: script/Arcueid/Day 6/06_00_ARC06_2_2_123.txt [hunk 5]
```diff
@@ -85,11 +86,11 @@ However, the woman in the lab coat halts in front of the door, then turns around
 // https://blog.goo.ne.jp/es2004/e/4b9473550e7138561787647f36e834cc
 // The seven years dross/dregs I interpret as seven years without exercising
 // his skills, sitting around growing stagnant.
-If, in future, you plan on getting hurt like this again, let me offer you a piece of advice.
+If, in the future, you plan on getting hurt like this again, let me offer you a piece of advice.
  Shiki-chan, your body isn't in peak condition, is it?
  So how about it?
  Don't you want to shake off the stagnation of these past seven years?
- Well, be warned that if you do it's reeeeally gonna hurt like hell though!"
+ Well, be warned that if you do, it's reeeeally gonna hurt like hell, though!"
```
[the remark this answers is not in the document]
Comment (Parzivus) [reply script/Arcueid/Day 6/06_00_ARC06_2_2_123.txt:93]: I think I'll take out the one after hell and keep the others.

File: script/Arcueid/Day 6/06_00_ARC06_4.txt [hunk 6]
```diff
@@ -199,7 +199,7 @@ parting ways.
 [sha:49a51e852448b924d12df3e580fc0b97a96389a7]{
 -- Page 336, Offset 14750.
 -- ……まったく情けない。あれだけ物思いに耽っておいて、まだ熱が冷めていないのか、俺は。
-...How pathetic. To sink this deeply into my reverie... my fever hasn't abated at all, has it.
+...How pathetic. To sink this deeply into my reverie... my fever hasn't abated at all, has it?
```
[the remark this answers is not in the document]
Comment (Parzivus) [reply script/Arcueid/Day 6/06_00_ARC06_4.txt:202]: It's alright to use a question mark for the rhetorical question here, I think. Could keep the period and strip off the "has it," but that might be drifting away from the JP sentence structure too much.

File: script/Arcueid/Day 6/06_00_ARC06_1.txt [hunk 7]
```diff
@@ -216,7 +217,7 @@ The cryptic words fly around inside my brain.
 [sha:746c752290f83469905518de92815163c2d9d502]{
 -- Page 16, Offset 13963.
 -- 思考を放棄したい誘惑に耐えながら、察すること10秒。
-Resisting the temptation to abandon the train of thought, I take 10 seconds to piece things together.
+Resisting the temptation to abandon the train of thought, I take ten seconds to piece things together.
```
Comment (Avuxo) [inline script/Arcueid/Day 6/06_00_ARC06_1.txt:220]: my train of thought

File: script/Arcueid/Day 6/06_00_ARC06_1_2.txt [hunk 8]
```diff
@@ -307,13 +308,13 @@ She has a dignified posture, a clear tone of voice, and eyes that never waver. E
 [sha:2380e70d4dfa6bebe9c8f70615956ff3ef0959d4]{
 -- Page 104, Offset 14177.
 -- がたん、とテーブルに足をひっかけて、秋葉は立ちあがった。
-// がたん、とテーブルに足をひっかけて、秋葉は立ちあがった。
-Akiha leaps to her feet, banging her shin against the table as she does so.
+// Seems a little superfluous
```
Comment (Avuxo) [inline script/Arcueid/Day 6/06_00_ARC06_1_2.txt:311]: what does? unless you mean the comment. then you can delete it, you're right. No need for an extra comment though

File: script/Arcueid/Day 6/06_00_ARC06_4.txt [hunk 9]
```diff
@@ -39,7 +39,7 @@ Akiha and I sit in the dining room.
 -- Page 326, Offset 14724.
 -- 琥珀さんお手製の夕食は、和牛をメインディッシュにした一連のコースだった。
 The dinner Kohaku-san had prepared for us consisted of a series of courses,
- culminating in a main course of wagyu beef.
+ culminating in a main dish of wagyu beef.
```
Comment (Avuxo) [inline script/Arcueid/Day 6/06_00_ARC06_4.txt:42]: メインディッシュ is just some wasei-eigo, it means "main course"

**Review by Avuxo — APPROVED** [review]
few comments but otherwise good stuff

**Comment by rschlaikjer** [comment]
Don't forget to update this PR with the suggested changes

**Comment by Parzivus** [comment]
I'll update and merge it tomorrow, should have time on the weekend

File: script/Arcueid/Day 6/06_00_ARC06_1_2.txt (hunk 8 above)
Comment (Parzivus) [reply script/Arcueid/Day 6/06_00_ARC06_1_2.txt:311]: I messed up saving this line, but it's pretty minor so I'll just leave as is

**Pull request merged**